assets/css/base.css

```diff
@@ -10,7 +10,7 @@ html {
 }
 body{
     font: 14px/24px "Microsoft YaHei","Arial",sans-serif;
-    background:url(/assets/images/beige_paper.png) left top repeat;
+    background:url(../../assets/images/beige_paper.png) left top repeat;

```

Comment:
We need to talk about this. I think there must be some misunderstanding between us.